### ISSUE TITLE
batch normalization could sync?

### ISSUE THREAD

**Issue by yjl9122** [uber/horovod#353]
If we can use syncbn?

**Comment by alsrgv** [comment]
@yjl9122, is this the same question as #349?  In short, global BN sync is possible with #331 and [TensorPack](https://github.com/tensorpack/tensorpack/blob/07783edb998cec3ec91c4312b39bd754cf9ececa/tensorpack/models/batch_norm.py#L226-L231).

**Comment by wangg12** [comment]
@alsrgv How about syncbn in PyTorch with horovod?

**Comment by tgaddair** [comment]
Hey @wangg12, see #1384.
